docs/specs/verification.md

```diff
@@ -0,0 +1,66 @@
+# User Verification
+
+## Definitions
+
+A user is verified if the user fulfill the condition specified by criteria.
+
+An identity is verifiable if:
+- it is an SSO identity; or
+- it is a login ID identity with a configured verifiable login ID key.
```

Comment:
So the developer can say, this login ID key is verifiable or not.

Something like this?

```yaml
identity:
  login_id:
    keys:
    - key: email
      type: email
      verifiable: false # By default it is true
```